tests/mypy/mypy-default.ini

```diff
@@ -5,7 +5,7 @@ warn_redundant_casts = True
 warn_unused_ignores = True
 disallow_any_generics = True
 check_untyped_defs = True
-no_implicit_reexport = True
+;no_implicit_reexport = True
```

Comment:
If we want to uncomment this line, to get the tests to pass, I'd need to change:
```python
from pydantic import BaseModel
```
to
```python
from pydantic.main import BaseModel
```
(Along with a couple other similar changes.)
Is this intentional?

Reply:
(Note: This mypy.ini is *not* the one used when running mypy over the project in CI / `make mypy`.)